### ISSUE TITLE
Test Case: QUEUE page/main - Please add all services to the Queue page

### ISSUE THREAD

**Issue by NYU-AMS** [codetodesign/NYUAMS#690]
Test Case: QUEUE page/main - Please Add all services to the Queue page

RP: 
FDM, PolyJet, BinderJet

3D Scan:
All in one list

**Comment by barryross** [comment]
How does `FDM, PolyJet, and BinderJet` relate to the two Rapid Prototyping devices in this list? 

<img width="882" alt="screen shot 2017-03-19 at 12 20 15 pm" src="https://cloud.githubusercontent.com/assets/2281667/24083963/7c4aa58c-0c9e-11e7-882c-18e51ea644eb.png">

**Comment by buck0land** [comment]
The connex name should be change to "polyjet".

The projet 660 are "binderJet".

The projet 7000 should be "SLA" 

FDM is new.
I holds to systems
Mojo and the Fortus 450

**Comment by leobardoramon** [comment]
@barryross , please wait on this until I provide you some wireframes.

**Comment by barryross** [comment]
@leobardoramon  👍

**Comment by barryross** [comment]
@buck0land thanks for the response

**Comment by leobardoramon** [comment]
*please note:

if renaming devices in Queue page will affect the speed of DEV greatly, we will put on HOLD.

@barryross , please keep us updated. ty

**Comment by barryross** [comment]
@leobardoramon @buck0land Is this what you guys are looking for but without reference to "laser" and with the updated names? 
<img width="597" alt="screen shot 2017-03-30 at 4 43 09 pm" src="https://cloud.githubusercontent.com/assets/2281667/24530771/152c35a4-1569-11e7-9227-8080adf9224c.png">
<img width="576" alt="screen shot 2017-03-30 at 4 43 13 pm" src="https://cloud.githubusercontent.com/assets/2281667/24530773/152db258-1569-11e7-8a01-85db338ada7d.png">

<img width="521" alt="screen shot 2017-03-30 at 4 43 16 pm" src="https://cloud.githubusercontent.com/assets/2281667/24530774/152fde48-1569-11e7-9233-343dbd7ab043.png">

**Comment by leobardoramon** [comment]
yes, just with updated names. ty @barryross

**Comment by barryross** [comment]
@leobardoramon great, thanks!

**Comment by barryross** [comment]
@leobardoramon I don't see the "ProJet 660" the @buck0land mentioned.  

Should I change "zPrinter 650" to "binderJet" ? 



<img width="996" alt="screen shot 2017-04-06 at 4 43 51 pm" src="https://cloud.githubusercontent.com/assets/2281667/24779917/48e40974-1ae8-11e7-8bdb-5d1ddf6bdc0e.png">

**Comment by barryross** [comment]
Getting there!

<img width="1198" alt="screen shot 2017-04-06 at 4 50 18 pm" src="https://cloud.githubusercontent.com/assets/2281667/24780033/267a7174-1ae9-11e7-9972-5da02439d2a6.png">

**Comment by leobardoramon** [comment]
@barryross , Projet 660 = Zprinter 650 , please change to "Binder Jet"

**Comment by barryross** [comment]
thank you, will do!

**Comment by barryross** [comment]
@leobardoramon I believe I have resolved this issue!

Relevant commit: 
https://github.com/codetodesign/NYUAMS/commit/603eacb56b4a94de6c3ca7b653dc26f340890d75

<img width="1430" alt="screen shot 2017-04-14 at 8 11 40 pm" src="https://cloud.githubusercontent.com/assets/2281667/25060307/a5502578-214e-11e7-9a67-192f4bc6cede.png">
<img width="1440" alt="screen shot 2017-04-14 at 8 11 45 pm" src="https://cloud.githubusercontent.com/assets/2281667/25060306/a54e4f82-214e-11e7-9923-f419baefe7ad.png">
<img width="1440" alt="screen shot 2017-04-14 at 8 11 48 pm" src="https://cloud.githubusercontent.com/assets/2281667/25060305/a54bf480-214e-11e7-9f8d-33c50f873db5.png">

**Comment by leobardoramon** [comment]
@barryross , I'm still noticing the old device names for RP service .

See screenshot :

<img width="1459" alt="screen shot 2017-06-14 at 3 50 25 pm" src="https://user-images.githubusercontent.com/2165796/27151707-72330890-5119-11e7-92a2-4656c18d9728.png">

**Comment by barryross** [comment]
@leobardoramon can you please follow the steps listed here , beneath 4/6/17 ? :https://github.com/codetodesign/NYUAMS/wiki/MySQL-Changes

**Comment by leobardoramon** [comment]
AhA!  Thanks @barryross !

see screenshot:
<img width="1106" alt="screen shot 2017-06-14 at 4 04 14 pm" src="https://user-images.githubusercontent.com/2165796/27152247-3eed68c0-511b-11e7-98c9-7d72ba85abdc.png">

**Comment by barryross** [comment]
welcome!
